syncd/NotificationProcessor.cpp

```diff
@@ -397,7 +398,16 @@ void NotificationProcessor::process_on_port_state_change(
          * switch vid.
          */
 
-        oper_stat->port_id = m_translator->translateRidToVid(oper_stat->port_id, SAI_NULL_OBJECT_ID);
+        oper_stat->port_id = m_translator->translateRidToVid(rid, SAI_NULL_OBJECT_ID);
+
+        if(oper_stat->port_id == SAI_NULL_OBJECT_ID)
+        {
+            /* TODO: Port might have already removed. Dont send the notification higher.
```

Comment:
if  you use multiline comment, follow style like other multiline comments

Reply:
ok